src/MLOps.NET.SQLServer.IntegrationTests/SQLServerMetaDataStoreTests.cs

```diff
@@ -0,0 +1,157 @@
+﻿using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using MLOps.NET.Entities;
+using MLOps.NET.SQLServer.Storage;
+using MLOps.NET.Storage;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace MLOps.NET.SQLServer.IntegrationTests
+{
+    [TestClass]
+    [TestCategory("IntegrationTestSqlServer")]
+    public class SQLServerMetaDataStoreTests
+    {
+        private const string connectionString = "Server=localhost;Database=MLOpsNET_IntegrationTests;Trusted_Connection=True;";
+        private SQLServerMetaDataStore sut;
+
+        [TestInitialize]
+        public void Initialize()
+        {
+            sut = new SQLServerMetaDataStore(new DbContextFactory(connectionString));
+        }
+
+        [TestCleanup]
+        public async Task TearDown()
+        {
+            var contextFactory = new DbContextFactory(connectionString);
+            var context = contextFactory.CreateDbContext();
+
+            var experiments = context.Experiments;
+            var runs = context.Runs;
+            var metrics = context.Metrics;
+            var hyperParameters = context.HyperParameters;
+
+            context.Experiments.RemoveRange(experiments);
+            context.Runs.RemoveRange(runs);
```

Comment:
Can  we do a EnsureDeleted here, by adding a new method in MLOPsContext , which internally calls Database.EnsureDeleted?

Reply:
I thought about it, but I'm hesitant at this point opening up a method on the DbContext that deletes the entire database. We may eventually end up there in the future, but I think some more fine-grained control is okay for now. I also think the tests may be a bit slower if we have to completely drop and re-create the database for each test, but I'm not 100% sure about that (for our purposes, it may actually be faster)

Reply:
@aslotte, we missed the confusionmatrix table here

Reply:
Again great catch, thanks! Pushed a change for it. I also tested using `EnsureDeletedAsync` just to see if it would slow it down or not. Using `EnsureDeletedAsync` all 7 integration tests would take 1 min 10 sec, while by just doing what we do now (cleaning up DbSets) they all take 10 sec totally so I think we're good with our current approach.